src/header/SluglineNav.tsx

```diff
@@ -2,47 +2,61 @@ import React from "react";
 import { Navbar, Nav } from "react-bootstrap";
 import { Link } from "react-router-dom";
 
-import "./SluglineNav.scss";
 import { useAuth } from "../auth/AuthProvider";
 
+import "./SluglineNav.scss";
+
+interface NavLinkProps {
+  to: string;
+  onClick?: () => void;
+  text: string;
+}
+
+// React Router's links and Bootstrap's navbar links don't play together too nicely,
+// so try and mash them together here
+const NavLink: React.FC<NavLinkProps> = props => {
+  const { text, ...rest } = props;
+  return (
+    <Nav.Item>
+      <Link {...rest} className="nav-link">
+        <h4>{props.text}</h4>
```

Comment:
For semantic reasons, we should probably style the links in CSS rather than using a heading element.